doc/api/stream.md

```diff
@@ -1143,6 +1143,15 @@ added: v12.3.0
 
 Getter for the property `objectMode` of a given `Readable` stream.
 
+##### readable.readableListening
```

Comment:
Nit: it seems this section needs to be placed before the `##### readable.readableObjectMode` section alphabetically.

Reply:
Good catch. Thanks for pointing it out. i will fix the order.